sunpy/visualization/animator/wcs.py

```diff
@@ -88,6 +91,11 @@ def __init__(self, data, wcs, slices, coord_params=None, ylim='dynamic', ylabel=
         self.ylim = ylim
         self.ylabel = ylabel
 
+        if clip_interval is not None and len(clip_interval) == 2:
```

Comment:
Is this check correct?

Reply:
Think there's an issue, gimme a second to verify

Reply:
Oh, sorry. I didn't see this